### PR TITLE
Priority bug fix

### PR DESCRIPTION
**Pull request by ArcLumin**
should stop reddit fucks from crashin the serb

### CODE REVIEW

**Comment by TheOneAndOnlyCreeperJoe** [comment]
changelog

credit me while I'm at it.

File: code/datums/wires/wires.dm [hunk 1]
```diff
@@ -126,6 +128,8 @@ var/list/wireColours = list("red", "blue", "green", "black", "orange", "brown",
 
 /datum/wires/Topic(href, href_list)
 	..()
+	if(wireused)
```
Comment (TheOneAndOnlyCreeperJoe) [inline code/datums/wires/wires.dm:131]: should be wire_used

File: code/datums/wires/wires.dm [hunk 2]
```diff
@@ -134,14 +138,20 @@ var/list/wireColours = list("red", "blue", "green", "black", "orange", "brown",
 			if(href_list["cut"]) // Toggles the cut/mend status
 				if(istype(I, /obj/item/weapon/wirecutters))
 					var/colour = href_list["cut"]
+					wireused = 1
```
Comment (TheOneAndOnlyCreeperJoe) [inline code/datums/wires/wires.dm:141]: should be wire_used

File: code/datums/wires/wires.dm [hunk 3]
```diff
@@ -134,14 +138,20 @@ var/list/wireColours = list("red", "blue", "green", "black", "orange", "brown",
...
 					CutWireColour(colour)
+					spawn(wire_cooldown)
+						wireused = 0
 				else
 					L << "<span class='warning'>You need wirecutters!</span>"
 
 			else if(href_list["pulse"])
 				if(istype(I, /obj/item/device/multitool))
 					var/colour = href_list["pulse"]
+					wireused = 1
```
Comment (TheOneAndOnlyCreeperJoe) [inline code/datums/wires/wires.dm:151]: should be wire_used

File: code/datums/wires/wires.dm [hunk 4]
```diff
@@ -134,14 +138,20 @@ var/list/wireColours = list("red", "blue", "green", "black", "orange", "brown",
...
 					PulseColour(colour)
+					spawn(wire_cooldown)
+						wireused = 0
```
Comment (TheOneAndOnlyCreeperJoe) [inline code/datums/wires/wires.dm:154]: should be wire_used

File: code/datums/wires/wires.dm [hunk 5]
```diff
@@ -160,7 +170,10 @@ var/list/wireColours = list("red", "blue", "green", "black", "orange", "brown",
 						if(A.attachable)
 							if(!L.drop_item())
 								return
+							wireused = 1
```
Comment (TheOneAndOnlyCreeperJoe) [inline code/datums/wires/wires.dm:173]: should be wire_used

File: code/datums/wires/wires.dm [hunk 6]
```diff
@@ -160,7 +170,10 @@ var/list/wireColours = list("red", "blue", "green", "black", "orange", "brown",
...
 							Attach(colour, A)
+							spawn(wire_cooldown)
+								wireused = 0
```
Comment (TheOneAndOnlyCreeperJoe) [inline code/datums/wires/wires.dm:176]: should be wire_used

File: code/datums/wires/wires.dm [hunk 7]
```diff
@@ -134,14 +138,20 @@ var/list/wireColours = list("red", "blue", "green", "black", "orange", "brown",
 			if(href_list["cut"]) // Toggles the cut/mend status
 				if(istype(I, /obj/item/weapon/wirecutters))
 					var/colour = href_list["cut"]
+					wireused = 1
 					CutWireColour(colour)
+					spawn(wire_cooldown)
+						wireused = 0
```
Comment (TheOneAndOnlyCreeperJoe) [inline code/datums/wires/wires.dm:144]: should be wire_used

**Review by TheOneAndOnlyCreeperJoe — COMMENTED** [review]
change shit

**Comment by ArcLumin** [comment]
hah

**Comment by TheOneAndOnlyCreeperJoe** [comment]
Code seems fine.

**Comment by Earthykiller127** [comment]
i'm in ur server crashing ur d00ds

**Comment by Carbonhell** [comment]
Succ my dicc pls

**Pull request merged**